library/common/config_template.cc

```diff
@@ -204,6 +204,9 @@ stats_flush_interval: {{ stats_flush_interval_seconds }}s
         - safe_regex:
             google_re2: {}
             regex: '^http.dispatcher.*'
+        - safe_regex:
+            google_re2: {}
+            regex: '^client.*'
```

Comment:
what rules do you have in mind? 
right now all the elements used for the stats are checked against the regex: `^[A-Za-z_]+\$`

Reply:
Talked with @junr03, long-term it might make sense to make people specify this in configuration, so that some thought goes into the stats they emit. For now, as a new, experimental interface, this is probably good enough to move forward.